src/components/FileUpload.js

```diff
@@ -33,7 +34,8 @@ class FileUpload extends Component {
     decrypt = (encrypted) => {
         try {
             let decrypted = this.xorWithKey(atob(encrypted), "key");
-            return JSON.parse(decrypted);
+            console.log(decrypted);
```

Comment:
You'll probably want to remove this from here as well.

Reply:
Same as above :)

I'll remove this one as well.

Reply:
Fixed.